docs/miscellaneous/take-screenshot.md

```diff
@@ -64,17 +68,17 @@ To take a screenshot of a specific area, you should use the following combinatio
 
 - **Hold down *Win* (the Windows button) and *Shift* and press *S***
 
-After you take a screenshot, it will be saved in the clipboard. In most cases you will be able to paste it into a document that you are currently editing by using *Ctrl + V* button combination. Alternatively, if you need to save the screenshot into a file, you should open the standard **Paint** program (or any other app that can work with images). Paste your screenshot there using the same button combination or by clicking the Paste button (usually in the top left corner of the screen) and then save it.
+After you take a screenshot, it will be saved in the clipboard. Usually, you will then be able to insert/paste it into, e.g., a document using the standard button combination *Ctrl + V*. Alternatively, if you need to save the screenshot into a file, you should open the standard **Paint** program (or any other app that can work with images). Paste your screenshot there using the same button combination or by clicking the Paste button (usually in the top left corner of the screen) and then save it.
```

Comment:
было норм, а вот e.g. стараемся не использовать

```suggestion
After you take a screenshot, it will be saved in the clipboard. Usually, you will then be able to paste it into a document using the standard button combination *Ctrl + V*. Alternatively, if you need to save the screenshot into a file, you should open the standard **Paint** program (or any other app that can work with images). Paste your screenshot there using the same button combination or by clicking the Paste button (usually in the top left corner of the screen) and then save it.
```